contrib/jfr_events/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id "java"
+    //    id "maven-publish"
+}
+
+description = 'OpenTelemetry Contrib JFR Events'
+ext.moduleName = "io.opentelemetry.contrib.jfr"
+
+sourceCompatibility = 11
+targetCompatibility = 11
+
+dependencies {
+    implementation project(':opentelemetry-api')
+    implementation project(':opentelemetry-sdk')
```

Comment:
Since this required the SDK should it be an SDK contribution instead? @jkwatson

Reply:
I think yes.